recipes/dcv_install.rb

```diff
@@ -106,31 +106,21 @@ def disable_lock_screen
 
     when 'ubuntu'
       apt_update
-      # Must install whoopsie separately before installing ubuntu-desktop to avoid whoopsie crash pop-up
-      package 'whoopsie' do
-        retries 3
-        retry_delay 5
-      end
-      # Install the desktop environment and the desktop manager packages
-      prereq_packages = %w[ubuntu-desktop mesa-utils]
-      package prereq_packages do
-        retries 10
-        retry_delay 5
-      end
-      # Must purge ifupdown before creating the AMI or the instance will have an ssh failure
-      package 'ifupdown' do
-        action :purge
-        retries 3
-        retry_delay 5
-      end
-      bash 'setup pre-req' do
+
+      bash 'install pre-req' do
         cwd Chef::Config[:file_cache_path]
+        # Must install whoopsie separately before installing ubuntu-desktop to avoid whoopsie crash pop-up
+        # Must purge ifupdown before creating the AMI or the instance will have an ssh failure
+        # Run dpkg --configure -a if there is a `dpkg interrupted` issue when installing ubuntu-desktop
         code <<-PREREQ
           set -e
+          apt -y install whoopsie
```

Comment:
Can we also set the env var for non interactive package installation? is it `DEBIAN_FRONTEND=noninteractive`?

Reply:
Done